packages/commons-ui-next/package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "next-commons-ui",
+	"version": "0.0.0",
+	"main": "index.js",
+	"license": "MIT",
+	"scripts": {
+		"test": "jest",
+		"lint": "eslint --fix './'",
+		"clean": "rm -rf .turbo node_modules",
+		"storybook": "start-storybook --ci -p 6006",
+		"build-storybook": "build-storybook --quiet -o dist"
+	},
+	"dependencies": {
+		"next": "12.1.0",
```

Comment:
Most of these should be `peerDependencies` ie shouldn’t be bundled with the package.

Reply:
Hmm...this peerDependencies and devDependencies kidogo zinanipiga chenga 🤦‍♀️  but yeah, will fix this @kilemensi

Reply:
Fixed